web/locales/en/plugin__netobserv-plugin.json

```diff
@@ -190,6 +188,7 @@
   "{{count}} Port(s)_plural": "{{count}} Port(s)",
   "{{count}} Protocol(s)": "{{count}} Protocol(s)",
   "{{count}} Protocol(s)_plural": "{{count}} Protocol(s)",
+  "The following metrics are incomplete. Add more filters or increase limit.": "The following metrics are incomplete. Add more filters or increase limit.",
```

Comment:
I am not sure that `incomplete` is clear enough for the user. Can I suggest this :
`Flow per request limit reached, following metrics can be inaccurate. Add more filters or increase limit.`

Reply:
Also, maybe say "Narrow down your search" rather than "Add filters", because it's not necessarily about adding filters: it could also be refining a regex (if a regex was used), using exact match, decreasing the time range ... "Narrow down your search" encompasses all of that

Reply:
Also, if "match any" is used, adding filters would worsen the situation :)

Reply:
So does this works for both of you `Flow per request limit reached, following metrics can be inaccurate. Narrow down your search or increase limit.` ?

Reply:
it works for me!